src/models/Movie.ts

```diff
@@ -0,0 +1,33 @@
+import Ids from '@/models/Ids';
+import FanartApi from '@/services/FanartApi';
+
+export default class Movie {
+    private readonly _title: string;
+    private readonly _year: number;
+    private readonly _ids: Ids;
+    private _thumbUrl: string = '';
+
+    constructor(newTitle: string, newYear: number, newIds: Ids) {
+        this._title = newTitle;
+        this._year = newYear;
+        this._ids = newIds;
+        FanartApi.instance.getMovieThumb(this._ids.imdb)
```

Comment:
This line may be overwriting the default `no-image.png` filepath in the `MediaTile`